### PR TITLE
chore: import mobile wallet and continue repo config

### PR DESCRIPTION
**Pull request by b0b4456**

### CODE REVIEW

**Comment by b0b4456** [comment]
@adilmezghouti PTAL

File: packages/core/src/storage-manager.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+// import AsyncStorage from '@react-native-async-storage/async-storage'
```
Comment (b0b4456) [inline packages/core/src/storage-manager.ts]: this shouldn't be in core...

File: packages/core/src/utils/coin-formatter.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import BigNumber from 'bignumber.js'
+import { unitToCurrency, assets } from '@liquality/cryptoassets'
+import { Asset } from '@liquality/cryptoassets/dist/src/types'
```
Comment (b0b4456) [inline packages/core/src/utils/coin-formatter.ts:4]: why isn't this type exported directly?

File: packages/core/src/abstract-wallet-manager.ts [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+      new BitcoinEsploraBatchApiProvider({
+        batchUrl: batchEsploraApi,
+        url: esploraApi,
+        network: bitcoinNetwork as BitcoinNetwork,
+        numberOfBlockConfirmation: 2
+      })
+    )
+
+    btcClient.addProvider(
+      // @ts-expect-error FIXME
```
Comment (b0b4456) [inline packages/core/src/abstract-wallet-manager.ts:84]: 💣

File: packages/core/src/storage-manager.ts (hunk 1 above)
Comment (adilmezghouti) [reply packages/core/src/storage-manager.ts]: Agreed. The logic that is mobile specific should live in the mobile app.

File: packages/core/src/utils/coin-formatter.ts (hunk 2 above)
Comment (adilmezghouti) [reply packages/core/src/utils/coin-formatter.ts:4]: This should be imported from the liquality types package which I did not exist until recently.

File: packages/core/src/abstract-wallet-manager.ts (hunk 3 above)
Comment (adilmezghouti) [reply packages/core/src/abstract-wallet-manager.ts:84]: Yes. This was put there as placeholder. Eventually, we need to have different implementations of the wallet based on the chain and use them through a strategy pattern. Or another pattern that helps us achieve the flexibility and modularity we need.

File: packages/core/src/utils/coin-formatter.ts (hunk 2 above)
Comment (b0b4456) [reply packages/core/src/utils/coin-formatter.ts:4]: I don't see `Asset` as an export of `@liquility/types`

Comment (adilmezghouti) [reply packages/core/src/utils/coin-formatter.ts:4]: If it does not, we then need to talk to the team working on the CAL to add it to the types repo or we can add it ourselves.

**Pull request merged**